src/systems/ec/implementations/vanilla/packages/ec-component-form-select/ec-component-form-select.js

```diff
@@ -184,6 +180,28 @@ export class Select {
     this.search.classList.add('ecl-text-input', 'ecl-text-input--m');
     this.search.setAttribute('type', 'text');
     this.search.setAttribute('placeholder', this.textSearch || '');
+    this.search.addEventListener('keyup', e => {
+      const keyword = e.target.value;
+      if (keyword.length > 2) {
+        this.checkboxes.forEach(checkbox => {
+          if (
+            !checkbox
+              .getAttribute('data-select-multiple-value')
```

Comment:
as a small improvement, we may ignore font case (in the example, searching for "bel" won't keep option "Belgium" enabled)